entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+# Create the user with ssh access
+useradd -d "/home/${SSH_USER}" -m -s "/bin/bash" "${SSH_USER}"
+mkdir -p /home/${SSH_USER}/.ssh/
+
+# Configure ssh_config
+mkdir -p /var/run/sshd
+if ! grep "PasswordAuthentication no" /etc/ssh/ssh_config ; then
+	echo "Port 22" >> /etc/ssh/ssh_config
+	echo "PasswordAuthentication no" >> /etc/ssh/ssh_config
+	echo "AuthenticationMethods 'publickey'" >> /etc/ssh/ssh_config
```

Comment:
@JesusZapata this is not needed because in the line before we just disabled the password auth